chia/timelord/timelord.py

```diff
@@ -88,9 +109,13 @@ def __init__(self, root_path, config: Dict, constants: ConsensusConstants):
         self.total_unfinished: int = 0
         self.total_infused: int = 0
         self.state_changed_callback: Optional[Callable] = None
-        self.sanitizer_mode = self.config["sanitizer_mode"]
+        self.bluebox_mode = self.config.get("bluebox_mode", False)
+        # Support backwards compatibility for the old `config.yaml` that has field `sanitizer_mode`.
+        if not self.bluebox_mode:
```

Comment:
it checks this directly on the next line:

`self.bluebox_mode = self.config.get("sanitizer_mode", False)`

This `bluebox_mode` replaces `sanitizer_mode` in `initial-config.yaml` (since we're blueboxing not sanitizing in the current terminology). So if `sanitizer_mode` is present in the old `config.yaml` and it's set to True, we'll still bluebox.